next-env.d.ts

```diff
@@ -0,0 +1,78 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
+
+interface FortyTwoCursusData {
```

Comment:
The file needs to be formatted base on `.prettierrc` config, you can use `Prettier` extension for `vscode`.

Reply:
Sorry I forgot to lint before submitting the merge request